### PR TITLE
new readme

### PR DESCRIPTION
**Pull request by NikolasKomonen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# VSCode XML Client
+
+
+## Get VSIX Extension
+
+* [Click here](http://download.jboss.org/jbosstools/vscode-xml/staging/) for the latests development VSIX
```
Comment (fbricon) [inline README.md]: latest development VSIX archives

Comment (fbricon) [inline README.md]: http://download.jboss.org/jbosstools/vscode-xml/staging/?C=M;O=D

**Comment by NikolasKomonen** [comment]
@fbricon updated

File: README.md [hunk 2]
```diff
@@ -1,9 +1,30 @@
 # VSCode XML Client
 
+## Description
+
+This VSCode extension provides support during the creation of XML documents
+
+## Features
```
Comment (fbricon) [inline README.md]: those are LSP features. Users don't care about those. You need to translate to plain English

File: README.md [hunk 3]
```diff
@@ -1,9 +1,30 @@
 # VSCode XML Client
 
+## Description
+
+This VSCode extension provides support during the creation of XML documents
```
Comment (fbricon) [inline README.md]: creation and edition.

Finish sentence with a period.

File: README.md [hunk 4]
```diff
@@ -1,9 +1,30 @@
...
+* [textDocument/documentSymbol](https://microsoft.github.io/language-server-protocol/specification#textDocument_documentSymbol).
+* [textDocument/documentHighlight](https://microsoft.github.io/language-server-protocol/specification#textDocument_documentHighlight).
+* [textDocument/completion](https://microsoft.github.io/language-server-protocol/specification#textDocument_completion).
+* [textDocument/hover](https://microsoft.github.io/language-server-protocol/specification#textDocument_hover).
+* [textDocument/rename](https://microsoft.github.io/language-server-protocol/specification#textDocument_rename).
+* [textDocument/foldingRanges](https://microsoft.github.io/language-server-protocol/specification#textDocument_foldingRange).
+
+## Requirements
+
+Java 8/9
```
Comment (fbricon) [inline README.md]: we (you) definitely need to fix Java 10 support (See #1)

File: README.md [hunk 5]
```diff
@@ -1,9 +1,30 @@
...
+
+Ensure Java path is set in either: 
+  
+  * `java.home` in VSCode preferences
+  * Environment variable `JAVA_HOME` or `JDK_HOME`
 
 ## Get VSIX Extension
 
-* [Click here](http://download.jboss.org/jbosstools/vscode-xml/staging/?C=M;O=D) for the latests development VSIX
+* [Click here](http://download.jboss.org/jbosstools/vscode-xml/staging/?C=M;O=D) latest development VSIX archives
```
Comment (fbricon) [inline README.md]: for the latest

File: README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  4) Select 'Install from VSIX...' and choose the VSIX file
+
+
+## Language Server for XML
+
+  The Language Server that this client uses for XML can be [found here](https://github.com/angelozerr/lsp4xml)
+
+## License
+
+  EPL 1.0, See LICENSE file.
```
Comment (fbricon) [inline README.md]: there is no LICENSE file, you need to add it

File: README.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+## Requirements
+
+  * Java 8/9
+
+  * Ensure Java path is set in either: 
+  
+    * `java.home` in VSCode preferences
+    * Environment variable `JAVA_HOME` or `JDK_HOME`
+
+## How to Install
```
Comment (fbricon) [inline README.md]: Installation

File: README.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+
+## Features
+
+  * Start and End Tag highlighting
+  * Code completion
+  * Document Folding
+  * Symbols and Renaming
+  * Error Validation
+
```
Comment (angelozerr) [inline README.md]: And Hover (based on XML Schema) & Format

File: README.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+  This VSCode extension provides creation and edition during the creation of XML documents.
+
+
+## Features
+
+  * Start and End Tag highlighting
+  * Code completion
+  * Document Folding
+  * Symbols and Renaming
+  * Error Validation
```
Comment (angelozerr) [inline README.md]: Syntax Error and XML Schema Validation

File: README.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+## Description
+
+  This VSCode extension provides creation and edition during the creation of XML documents.
+
+
+## Features
+
+  * Start and End Tag highlighting
+  * Code completion
```
Comment (angelozerr) [inline README.md]: Code completion based on XML Schema Validation

**Comment by NikolasKomonen** [comment]
@fbricon updated with angelo's suggestions, and consideration for all java versions

File: src/requirements.ts [hunk 11]
```diff
@@ -79,9 +79,17 @@ function readJavaConfig() : string {
 function checkJavaVersion(java_home: string): Promise<number> {
     return new Promise((resolve, reject) => {
         cp.execFile(java_home + '/bin/java', ['-version'], {}, (error, stdout, stderr) => {
```
Comment (fbricon) [inline src/requirements.ts]: I'll allow the quick n'dirty fix for now, but you should make it more robust, see #1

File: README.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
+# VSCode XML Client
+
+## Description
+
+  This VSCode extension provides creation and edition during the creation of XML documents.
```
Comment (fbricon) [inline README.md]: creation x 2

File: README.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+    * `java.home` in VSCode preferences
+    * Environment variable `JAVA_HOME` or `JDK_HOME`
+
+## Language Server for XML
+
+  The Language Server for this client can be [found here](https://github.com/angelozerr/lsp4xml).
+
+## License
+
+  EPL 1.0, See [LICENSE](https://github.com/NikolasKomonen/vscode-xml/blob/readme/LICENSE) file.
```
Comment (fbricon) [inline README.md]: [LICENSE](https://github.com/redhat-developer/vscode-xml/blob/readme/LICENSE)

**Pull request merged**